db/warehouse/migrate/20230613190449_adjust_external_ids_uniq_idx.rb

```diff
@@ -0,0 +1,14 @@
+class AdjustExternalIdsUniqIdx < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+  def change
+    # allow more than one client MCI ID
+    add_index :external_ids,
+        [:source_id, :source_type, :remote_credential_id],
+        unique: true,
+        name: 'uidx_external_ids_source_value',
+        where: "(namespace <> 'ac_hmis_mci')",
+        algorithm: :concurrently
```

Comment:
So this is saying, every namespace EXCEPT mci ids must have no more than 1 value per remote credential per client? If we run into another ExternalID that allows multiple values, how would we handle that? I don't know how likely that is, so maybe this is fine for now.

Reply:
> So this is saying, every namespace EXCEPT mci ids must have no more than 1 value per remote credential per client? If we run into another ExternalID that allows multiple values, how would we handle that? I don't know how likely that is, so maybe this is fine for now.

Yes, that's right. If we need another namespace that is non-unique, we could expand this condition or perhaps we add a dedicated flag. 

Looking at it again I realize that namespace can be null which doesn't match the condition 🤦 - I'll adjust that